force-app/main/default/lwc/templateTableDetails/templateTableDetails.css

```diff
@@ -0,0 +1,21 @@
+@media print {
```

Comment:
This LWC will have the styles from the local CSS file other than Static Resource, is that a case of no CSS revisions later?